client/src/pages/FloorPlan.js

```diff
@@ -22,11 +22,19 @@ const Desk = ({ desk }) => {
 			<text fontSize={40} x={64} y={90}>
 				{desk.id}
 			</text>
+			<text fontSize={25} x={20} y={140}>
+				{bookingForThisDesk?.name}
+			</text>
 		</>
 	);
 };
 
-const FloorPlan = ({ desks }) => {
+const FloorPlan = ({ desks, bookingsByDate }) => {
```

Comment:
I think this should be just bookings. Then a FloorPlan can show any bookings it gets told to show, without worrying what they are (this is only a change to the variable name)